paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -961,6 +961,30 @@ class ReductionMKLDNNHandler
 
     this->AcquireForwardPrimitiveDescriptor(algo, x_md, y_md, p, eps);
   }
+
+  ReductionMKLDNNHandler(const dnnl::algorithm algo, const float p,
```

Comment:
Both ReductionMKLDNNHandler constructors differ by only attr param. Could you make one of them to be using/calling the second one in order to keep it DRY, please? What is you opinion ?

Reply:
You're right, I didn't think of that. I'll make the new ReductionMKLDNNHander set a default value (NULL) to the attr parameter like this
```suggestion
  ReductionMKLDNNHandler(
      const dnnl::algorithm algo, const float p, const float eps,
      const mkldnn::engine engine, platform::Place cpu_place, const Tensor* x,
      const Tensor* y, std::vector<int64_t> y_tz,
      const dnnl::primitive_attr& attr = NULL)
```